### ISSUE TITLE
Bento: Components missing CSS

### ISSUE THREAD

**Issue by swissspidy** [ampproject/amphtml#35413]
### Description

When importing a React component such as `@ampproject/amp-base-carousel`, there seems to be no way to get the component's CSS as originally defined in [`component.jss.js`](https://github.com/ampproject/amphtml/blob/65a3b3c8c7583bbe5e559fab0a04b14b76626d6b/extensions/amp-base-carousel/1.0/component.jss.js).

So while I get a nicely styled component when using the `<amp-base-carousel>` custom element, in React I get a completely unstyled carousel with class names generated by JSS, but no actual styles for them.

And `amp-base-carousel.css` only contains pre-upgrade styles, so that doesn't help either.

How do I get the CSS?

### Reproduction Steps

I am basically doing this:

```js
import { BaseCarousel } from '@ampproject/amp-base-carousel/react';

function SomeComponent() {
	  return (
		    <BaseCarousel
		    autoAdvance={ true }
		    loop={ true }
		    snap={ true }
		    >
	  );
}
```

### AMP Version Affected

2107092322000

**Comment by samouri** [comment]
I believe we have two options:
1. disable the transform [babel-plugin-transform-jss](https://github.com/ampproject/amphtml/blob/main/build-system/babel-plugins/babel-plugin-transform-jss/index.js) for npm builds, so that the jss hooks stay in place. This requires the bundle contains the jss package.
2. keep the transform enabled, but start distributing the associated CSS files. In this case, the pubs need to include the CSS on the page manually, but jss does not need to be part of the bundle

**Comment by swissspidy** [comment]
From a developer perspective I'd prefer (2) just so I have more flexibility with how the CSS is being added to the page.

**Comment by kristoferbaxter** [comment]
I'd also push toward #2.

**Comment by caroqliu** [comment]
Fixed by #35446
